examples/all/manifests/thanos-compactor-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: thanos-compactor
+  name: thanos-compactor
+  namespace: monitoring
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: thanos-compactor
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: thanos-compactor
+    spec:
+      containers:
+      - args:
+        - compact
+        - --wait
+        - --retention.resolution-raw=16d
```

Comment:
Is that what we want? 16d only?

Reply:
This is just a simple starting point for people to tweak it later on to their exact needs.